go/doc.go

```diff
@@ -0,0 +1,22 @@
+// Package hyperkit provides a Go wrapper around the hyperkit
```

Comment:
> As is customary in go when a package has several files.

There are three files here `hyperkit.go` and `pty_util_{darwin,fallback}.go`. Keeping the docs in `hyperkit.go` makes sense given the peripheral nature of the other files. Please drop this commit.

Reply:
Hi @ijc.  Thanks for the review.

Okay.  I'm also creating more files, with log.go and disk.go, and `doc.go` is the common practice I saw.  But I don't care and sure can drop that.